### PR TITLE
Add RUM integration for Android

### PR DESCRIPTION
**Pull request by milena-markovic**
### What does this PR do?

Adds Datadog RUM integration tile for Android apps.

### Motivation

Surfacing the existing Android-RUM integration

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

- [x] Logo alias ready in druids

### CODE REVIEW

File: android/manifest.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "metric_prefix": "android.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "",
+  "guid": "a70b6926-49a8-4f90-8190-315170e97e4f",
+  "support": "contrib",
+  "supported_os": [],
+  "public_title": "Android",
+  "categories": [
+    ""
```
Comment (hithwen) [inline android/manifest.json:15]:
```suggestion
    "log collection"
```

**Comment by kayayarai** [comment]
Opened https://datadoghq.atlassian.net/browse/DOCS-3655 to track editorial review

**Review by alai97 — APPROVED** [review]

File: android/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+Need help? Contact [Datadog Support][7].
+
+## Further Reading
+
+Additional helpful documentation, links, and articles:
+
+- [Android and Android TV Monitoring][8]
+
+[1]: https://app.datadoghq.com/integrations/android
+[2]: https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces?tab=androidrum#setup-rum
```
Comment (alai97) [inline android/README.md:62]:
```suggestion
[2]: https://docs.datadoghq.com/real_user_monitoring/android/?tabs=kotlin#setup
```

**Review by hithwen — CHANGES_REQUESTED** [review]
> The `validations` job has failed; please review the `Files changed` tab for possible suggestions to resolve.

Please, look at and address the validation failures

**Comment by milena-markovic** [comment]
> > The `validations` job has failed; please review the `Files changed` tab for possible suggestions to resolve.
> 
> Please, look at and address the validation failures

My bad - I thought it had passed when I re-requested review. I've fixed one, but I'm not sure how to go about the error
`The tile description is 73 characters long. It should be no longer than 80 characters.` since it's already shorter than 80 characters?

**Comment by hithwen** [comment]
I'm also thinking we should probably rename the folder and integration_id to android_rum, just in case we have another sort of android integration in the fugure

**Review by hithwen — APPROVED** [review]

**Pull request merged**